modules/op-installing-tkn-on-linux.adoc

```diff
@@ -14,11 +14,13 @@ For Linux distributions, you can download the CLI directly as a `tar.gz` archive
 
 . Download the relevant CLI.
 
-* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.23.1/tkn-linux-amd64-0.23.1.tar.gz[Linux (x86_64, amd64)]
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.24.0/tkn-linux-amd64-0.24.0.tar.gz[Linux (x86_64, amd64)]
```

Comment:
```suggestion
* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.24.0/tkn-linux-amd64.tar.gz[Linux (x86_64, amd64)]
```
the filename for this (and other architectures in the PR) will be different from this time on.